scripts/hera-sim-vis.py

```diff
@@ -165,10 +165,14 @@ def cprint(*args, **kwargs):
         outfile = (
             base_path / f"{cfg_filing['outfile_name']}.{cfg_filing['output_format']}"
         )
+        if "clobber" not in cfg_filing:
+            clobber = False
```

Comment:
Maybe also print a warning here that `clobber` is being set to `False`.

Reply:
I don't think we'll need the warning. If clobber is False, and the file already exists, then an error will be raised later anyway, right? I think setting to False is non-destructive and so doesn't need a warning.

Reply:
To make the code slightly simpler, could remove all this if/else, and just replace the line 171 with `clobber=cfg_filing.get('clobber', False)`

Reply:
Yes, pyuvdata raises an error if the file exists and clobber is set to False.  And I forgot about the get function.  I'll make that change.